task-3/results/app/export_data.py

```
@@ -0,0 +1,259 @@
+            'password': os.environ.get('DB_PASSWORD', 'postgres')
+        }
+
+        self.s3_config = {
+            'endpoint': os.environ.get('S3_ENDPOINT', 'http://minio:9000'),
+            'access_key': os.environ.get('S3_ACCESS_KEY', 'minioadmin'),
+            'secret_key': os.environ.get('S3_SECRET_KEY', 'minioadmin'),
+            'bucket': os.environ.get('S3_BUCKET', 'analytics-data')
+        }
+
+        self.export_date = datetime.now().date()
+        self.export_path = f"/tmp/export_{self.export_date}"
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/export_"

(S108)